public/app/core/utils/explore.ts

```diff
@@ -107,7 +108,8 @@ export function buildQueryTransaction(
   queryOptions: QueryOptions,
   range: TimeRange,
   scanning: boolean,
-  timeZone?: TimeZone
+  timeZone?: TimeZone,
+  scopedVars?: ScopedVars
```

Comment:
I don't see it being passed when the query is run 🤔

Reply:
Besides a bug with the dirty state modals, I see the interpolation working. This is between two test datasources, source with logs, target with CSV content with nothing but one of the variables. What's not working?